vignettes/mmrm_review_methods.Rmd

```diff
@@ -0,0 +1,104 @@
+## Data Sets
+
+The FEV data set is contains measurements of FEV1 (forced expired volume in one second), a measure of how quickly the lungs can be emptied. Low levels of FEV1 may indicate chronic obstructive pulmonary disease (COPD). This data set is used to illustrate model fitting with the `mmrm`, `lme4`, `nlme`, `glmmtmb` R packages as well as `SAS`.
+
+The FEV data set
+
+```{r, eval=FALSE}
+tableone::tableone(fev_data) # something like this to illustrate fev_data
+```
+
+## Models
+
+### Unstructured Covariance
+
+#### SAS
+```{sas, eval=FALSE}
+PROC MIXED DATA = fev_data cl method=reml;
+CLASS RACE(ref = 'Asian') AVISIT(ref = 'VIS1') SEX(ref = 'Male') ARMCD(ref = 'PBO') USUBJID;
+MODEL FEV1 = ARMCD AVISIT ARMCD*AVISIT RACE SEX / ddfm=satterthwaite solution chisq;
+REPEATED AVISIT / subject=USUBJID type=un r rcorr;
+```
+#### mmrm
+```{r, eval=FALSE}
+mmrm::mmrm(formula = FEV1 ~ RACE + SEX + ARMCD * AVISIT + us(AVISIT | USUBJID),
+  data = fev_data)
+```
+#### nlme
+```{r, eval = FALSE}
+nlme::gls(formula = FEV1 ~ RACE + SEX + ARMCD * AVISIT,
+          data = fev_data,
+          correlation = nlme::corSymm(form = ~AVISIT | USUBJID),
+          weights = nlme::varIdent(form = ~1|AVISIT),
+          na.action = na.omit)
+```
+#### lme4
```

Comment:
lme4 scope vs mmrm (no explicit covariance structure?)

Reply:
Can you clarify what you are trying to ask here?

Reply:
Sure, I thought it might be good to mentioned here what lme4 can NOT do that we are interested in (cov structures).

Reply:
Thanks! Yeah this is still quite a work in progress, will definitely end up with a summary of what can/can't be done.